scripts/combine_zarr.py

```diff
@@ -128,8 +136,12 @@ def setup_input(samples, input_pattern, seqid, field):
     input_paths = [input_pattern.format(sample=s) for s in samples]
     input_stores = [zarr.ZipStore(ip, mode='r') for ip in input_paths]
     input_roots = [zarr.group(store) for store in input_stores]
-    input_arrays = [root[s][seqid]['calldata'][field] for root, s in zip(input_roots, samples)]
+    input_arrays = [root[s][seqid][field] for root, s in zip(input_roots, samples)]
     input_arrays = [da.from_array(a, chunks=a.chunks) for a in input_arrays]
+
+    # here we add a dim to allow the hstack to work. must share the shape (X, 1, )
+    input_arrays = [a.reshape((-1, 1)) if a.ndim < 2 else a for a in input_arrays]
```

Comment:
```suggestion
    input_arrays = [a[:, None] if a.ndim == 1 else a for a in input_arrays]
```

Has the same effect but possibly slightly clearer that a new dimension is being inserted.